test/spec/ApiConfigurationSpec.ts

```diff
@@ -0,0 +1,280 @@
+import { DefaultApiConfigurationParser } from "configuration/apiConfigurationParser";
+import { SubscriberEventType } from "configuration/apiConfiguration";
+
+const xml = `<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Finally I put the xml in a separate file. The config is already minimalist (based on a simple api)